docs/custom_mutators.md

```diff
@@ -4,6 +4,9 @@ This file describes how you can implement custom mutations to be used in AFL.
 For now, we support C/C++ library and Python module, collectivelly named as the
 custom mutator.
 
+There is also experimental rust support in `custom_mutators/rust`.
+Please refer to this directory for documentation.
```

Comment:
"... by executing `cargo doc`..." or something like this. some people might want to pick up rust and not know exactly how rustdoc works. like me :)

Reply:
Added a note about it! Thanks!